keiko-mem-spring/keiko-mem-spring.gradle

```diff
@@ -15,6 +15,6 @@ dependencies {
 
 test {
   useJUnitPlatform {
-    includeEngines "jupiter"
+    includeEngines "junit-jupiter"
```

Comment:
Does it work to remove includeEngines altogether?  That's what happens in [build.gradle](https://github.com/spinnaker/orca/blob/968da95d56b8d9a59c034beeab1d97b9e6200440/build.gradle#L73).

Reply:
And because`useJUnitPlatform` is in the top-level build.gradle for subprojects, I bet it works to remove the whole test block.

Reply:
Verified by removing the `test { }`. Test executed and passed.